src/public/modules/forms/helpers/CsvMergedHeadersGenerator.ts

```diff
@@ -147,7 +147,9 @@ export class CsvMergedHeadersGenerator extends CsvGenerator {
   ): string {
     const fieldRecord = unprocessedRecord[fieldId]
     if (!fieldRecord) return ''
-    return fieldRecord.getAnswer(colIndex)
+    const answer = fieldRecord.getAnswer(colIndex)
+    // wrap answer in quotes if answer is not a number
+    return isNaN(Number(answer)) ? `'${answer}'` : answer
```

Comment:
Could I please check - why can't we wrap in quotes even if it is a number?

Reply:
thought I would follow the suggestion in the [issue](https://github.com/datagovsg/formsg-private/issues/133) and leave numbers as they are, so that they can be manipulated in excel if the users wish. Also thought it might be possible that current users run excel macros/formulas on existing forms. it would definitely be more straightforward if we wrapped everything in quotes though! wdyt? :)

Reply:
thanks for highlighting! i think it is fine then. just a minor suggestion for documentation
```suggestion
    const answer = fieldRecord.getAnswer(colIndex)
    // wrap answer in quotes if answer is not a number. 
    // for number, do not wrap in quotes so that excel parses it as number
    return isNaN(Number(answer)) ? `'${answer}'` : answer
```